frontend/server/src/main/java/com/amazonaws/ml/mms/http/InvalidPluginException.java

```diff
@@ -0,0 +1,17 @@
+package com.amazonaws.ml.mms.http;
+
+public class InvalidPluginException extends RuntimeException {
+
+    static final long serialVersionUID = 1L;
+    /**
+     * Constructs an {@code MethodNotAllowedException} with {@code null} as its error detail
+     * message.
+     */
+    public InvalidPluginException() {
+        super("Registered plugin is invalid. Please re-check the configuration and the plugins.");
```

Comment:
This isn't a good idea.

Reply:
You mean the message is bad? I can make it more generic.

Reply:
I mean adding message in default contractor of Exception isn't good idea, user should explicitly call new InvalidPluginException(xxx)

Reply:
This is not available to users. Users only have ModelServingEndpointException through SDK.